.github/workflows/precommits_check.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.9"
+          python-version: "3.11"
```

Comment:
If we're going to use `3.11` as developers in our dev containers, we should probably upgrade the CI versioning as well.